antibodies/instance_analysis_workflow1.py

```diff
@@ -117,6 +120,13 @@ def parse_instance_config1():
     default_scale_factors = [1, 2, 4, 8]
     parser.add("--scale_factors", default=default_scale_factors)
 
+    # tagged outliers from a given plate
+    # if plate_name is empty we will try to infer it from the 'input_folder' name
+    parser.add("--plate_name", default=None, nargs='+', type=str, help="The name of the imaged plate")
```

Comment:
Do we have to add this as a command line argument?
We can usually parse this from the folder name, that would be easier than adding more command line options.

Reply:
Ok, nvm I just read the comment above :D